app/design/frontend/Edu/theme.xml

```diff
@@ -0,0 +1,4 @@
+<theme xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../lib/internal/Magento/Framework/Config/etc/theme.xsd">
+    <title>Edu Ultimate</title>
+    <parent>Magento/blank</parent>
+</theme>
```

Comment:
Missing new line

Reply:
I tried to do like Yandex - bem, https://ru.bem.info/methodology/quick-start/